tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/CassandraSinkTester.java

```diff
@@ -60,7 +60,7 @@ public static CassandraSinkTester createTester(boolean builtin) {
     private Session session;
 
     private CassandraSinkTester() {
-        super(NAME, "/pulsar/connectors/pulsar-io-cassandra-2.2.0-incubating-SNAPSHOT.nar", "org.apache.pulsar.io.cassandra.CassandraStringSink");
+        super(NAME, "/pulsar/connectors/pulsar-io-cassandra-2.4.0-SNAPSHOT.nar", "org.apache.pulsar.io.cassandra.CassandraStringSink");
```

Comment:
We should construct this string with the current version, with `PulsarVersion.getVersion()`

Reply:
Fixed